java/example_code/cloudmap/src/main/java/aws/example/cloudmap/CreateService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author : Rajib Deb
+ * 
+ * This program creates a service in the given namespace in AWS Cloud Map
+ * Please replace the namespace ID with the id that you are using for your project
+ */
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// This file is licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License. A copy of
+// the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+// snippet-sourcedescription:[DiscoverInstanceRequest.java demonstrates how to ....]
```

Comment:
Please update the file name, and complete the sentence to describe what this code sample is expected to do. 😸

Reply:
This is done